art/ryangolsing/cup.html

```diff
@@ -0,0 +1,126 @@
+  <div id="container">
+    <div class="cup">
+      <span class="heart"></span>
+      <div class="handle"></div>
+      <div class="steam">
+        <ul>
+          <li></li>
+          <li></li>
+          <li></li>
+          <li></li>
+          <li></li>
+        </ul>
+      </div>
+    </div>
+  </div>
+
+<style>
+	#container {
+  background-color: #696d7d;
+  border-top: 500px solid #424242;
```

Comment:
hey, this animation is great!
the only problem is that i show all the art with a width of 350px and a height of 300px, so the border-top needs to be around 10px.